features/index.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Landing Page content
+  As a visitor
+  in order to access the app
+  I need to see an index page
+  I would like to be presented with some examples of restaurants that deliver food
```

Comment:
Not a blocker though. Please update in Pivotal Tracker and read up on the User Story format.